cd/application-deployment/staging/vaec-celery-task-definition.json

```diff
@@ -212,6 +212,10 @@
         {
           "name": "COMP_AND_PEN_DYNAMODB_NAME",
           "value": "staging-bip-payment-notification-table"
+        },
+        {
+          "name": "VA_PROFILE_V3_USE_PROFILE_API_V3",
+          "value": "False"
```

Comment:
also new.  The check for the state of the feature flag defaults to False if the entry isn't present, but it's probably best to be explicit about it.